### PR TITLE
maybe this will help to run the colab again

### PR DESCRIPTION
**Pull request by Siddd25**

### CODE REVIEW

**Comment by v-iashin** [comment]
Ok, you were the first. I also created the same PR: #56

**Comment by v-iashin** [comment]
Can you remove torchvision as it is not required really?

File: conda_env.yml [hunk 1]
```diff
@@ -78,6 +78,7 @@ dependencies:
   - wrapt=1.10.11=py37h14c3975_2
   - xz=5.2.5=h7b6447c_0
   - zlib=1.2.11=h7b6447c_3
+  - torchvision=0.4.0=cuda100py37hecfc37a_0
```
Comment (v-iashin) [inline conda_env.yml:81]: no need

**Comment by Siddd25** [comment]
will the colab run successfully now @v-iashin ?!

**Comment by v-iashin** [comment]
fixed in #58. Could you try running it?

**Pull request merged**